packages/components/src/Toast/Toast.css

```diff
@@ -0,0 +1,50 @@
+:root {
+  --toast-shadow: 0 var(--space-base) var(--space-base) rgba(0, 0, 0, 0.16),
+    0 0 var(--space-small) rgba(0, 0, 0, 0.12);
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: fixed;
+  bottom: var(--space-large);
+  left: 50%;
+  z-index: 10000;
```

Comment:
Would probably be good to use or add one of these:

packages/design/src/Elevations.mdx